docs/docs/reference/custom-components/05-aica-package-toml.md

```diff
@@ -301,6 +362,18 @@ Some examples:
 source = "build-context://my_source"
 ```
 
+##### Version constraints
+
+Version constraints follow the syntax of the [Terraform version constraints](https://www.terraform.io/docs/language/expressions/version-constraints.html) which is similar to syntax used by NPM, yarn or pip. Here is a quick summary:
+
+- Versions constraints are composed of one or more conditions separated by commas, e.g. `>= 1.0.0, < 2.0.0`.
+- Each version specified must be a valid [semver](https://semver.org/) version, e.g. `1.0.0`.
+- The following operators are supported:
+  - `=` or no operator: allow only the exact version, cannot be combined with other conditions.
+  - `!=`: exclude a specific version.
+  - `>`, `>=`, `<`, `<=`: comparison against a specific version, allowing any version matching the operator. `>` allows newer version and `<` allows older version.
+  - `~>`: allows only the right most number of the version to increase. This is useful to allow only patch or minor versions to increase, e.g. `~> 1.0` allows `1.1`, `1.2`, etc. but not `2.0` and `~> 1.0.0` allows `1.0.1`, `1.0.2`, etc. but not `1.1.0`.
+
 ### Examples
 
 #### Basic setup
```

Comment:
The examples are still on `#syntax=ghcr.io/aica-technology/package-builder:v0.0.13`, wouldn't you prefer to update those too?

Reply:
This is crazy, I went through every page multiple times, did search and everything and somehow I still missed 4 of them? 😂